components/Votes/Votes.tsx

```diff
@@ -212,6 +214,10 @@ export function Votes() {
     }
     return actionConfig;
   }
+
+  const signatureExplanationText =
+    "UMA uses this signature to verify that you are the owner of this address. We must do this to prevent double voting.";
```

Comment:
"UMA uses this signature to verify that you are the owner of this address. This allows us to hide your vote during the commit phase and decode it during the reveal phase."

@mrice32 @chrismaree might want to double check this copy

Reply:
ye, we dont need to worry too much about the copy as @smb2796 has this other copy https://github.com/UMAprotocol/voter-dapp-v2/pull/138 PR that we aim to fix all the copy in one go. that'll be worked on this week.